src/test/resources/Tests/audio/playlist.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <playlist>
```

Comment:
Should this be used in the unit test?

Reply:
this code doesn't deal with `playlist.xml`. I've changed this file because I've realised its content is wrong and potentially misleading. So fixed it.
Sorry if this confused you

Reply:
oh ok